README.md

```diff
@@ -54,12 +54,28 @@ Other documentation can be found in [`docs` directory](docs).
 
 ## Support
 
-If you encounter any problems with this release please fill out the
-[PROBLEM-REPORT-FORM](PROBLEM-REPORT-FORM) file found in this directory and use
-it when posting to the [mailing list](http://opendds.org/support.html) or
-creating a [GitHub Issue](https://github.com/objectcomputing/OpenDDS/issues).
+For commercial support, please see https://opendds.org/support.html.
 
-For commercial support please see https://opendds.org/support.html.
+Questions concerning OpenDDS should be directed to [GitHub Discussions](https://github.com/objectcomputing/OpenDDS/discussions) or the [mailing list](http://opendds.org/support.html).
+
+If you wish to file a bug report:
+
+1. [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) OpenDDS on GitHub.
+2. Add a [minimum working example](#mwe) that demonstrates the problem.
+3. Create a pull request.
+4. Fill out the [PROBLEM-REPORT-FORM](PROBLEM-REPORT-FORM) and attach it to the pull request.
+
+### <a id="mwe">How to make a minimum working example</a>
+
+1. Create a directory `tests/DCPS/MyExample` (`MyExample` is a placeholder.)
+2. Add the IDL, code, and configuration files for the example programs.  Be sure to exclude any proprietary information as the submitted example will be public.
+3. Add an MPC file that builds the test programs.
+4. Add a launcher script `run_test.pl`.  Be sure to document the options.  See `tests/DCPS/HelloWorld/run_test.pl` for inspiration.
+5. Add a `README.rst` file that gives a high-level overview of the example and the problem that it illustrates.
```

Comment:
Do we want them to add the test to a list file so it runs on the CI?

Reply:
I would say "yes" but only if it was fool-proof.  As is, you have to understand what goes in the list file and when you should use one list versus another.

Reply:
We do sort of have documentation https://opendds.readthedocs.io/en/master/internal/running_tests.html#manual-configuration, but yea even if that was improved on, I agree it's still not straightforward. Also I guess that documentation is focused on running the tests and not putting the entry in the list file.